ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkTable/PXEColumn/PXEColumn.tsx

```diff
@@ -0,0 +1,35 @@
+import { Icon } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+
+import machineSelectors from "app/store/machine/selectors";
+import type {
+  Machine,
+  NetworkInterface,
+  NetworkLink,
+} from "app/store/machine/types";
+import { isBootInterface } from "app/store/machine/utils";
+import type { RootState } from "app/store/root/types";
+
+type Props = {
+  link?: NetworkLink | null;
+  nic?: NetworkInterface | null;
+  systemId: Machine["system_id"];
+};
+
+const PXEColumn = ({ link, nic, systemId }: Props): JSX.Element | null => {
+  const machine = useSelector((state: RootState) =>
+    machineSelectors.getById(state, systemId)
+  );
+  if (!machine) {
+    return null;
+  }
+  const isBoot = isBootInterface(machine, nic, link);
+
+  return isBoot ? (
+    <span className="u-align--center">
+      <Icon name="success" />
```

Comment:
I think we're trying to avoid using the success icon now, unless it's specifically for a successful action or something similar (test passed, form submitted, etc). There's a MAAS-specific icon `p-icon--tick` that we've used elsewhere that I think would make more sense here.

Reply:
Done.